pkg/descheduler/apis/config/types.go

```diff
@@ -77,6 +77,7 @@ type DeschedulerProfile struct {
 	Name         string
 	PluginConfig []PluginConfig
 	Plugins      *Plugins
+	NodeSelector *metav1.LabelSelector
```

Comment:
Shouldn't this be a plugin configuration?

Reply:
Generally speaking Yes. And actually I recommend plugins to support configuration of multiple `NodeSelector` like what `LowNodeLoad` do with `NodePool`. This PR is intended to solve the kubernetes descheduler plugin adaption problem for multiple node configuration.